azurelinuxagent/common/cgroupapi.py

```diff
@@ -34,6 +35,7 @@
 VM_AGENT_CGROUP_NAME = "walinuxagent.service"
 EXTENSIONS_ROOT_CGROUP_NAME = "walinuxagent.extensions"
 UNIT_FILES_FILE_SYSTEM_PATH = "/etc/systemd/system"
+DAEMON_PID_FILE = "/var/run/waagent.pid"
```

Comment:
the file can be changed in waagent.conf - let's use get_agent_pid_file_path instead of hardcoding the path

Reply:
Missed that function, thanks.